docs/source/configuration/migration.md

```diff
@@ -0,0 +1,56 @@
+## Migration guide for config loaders
+The `ConfigLoader` and `TemplatedConfigLoader` classes are now deprecated and scheduled for removal in Kedro `0.19.0`. To ensure smooth transitions for users, we strongly recommend adopting the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) promptly.
```

Comment:
```suggestion
The `ConfigLoader` and `TemplatedConfigLoader` classes have been deprecated since Kedro `0.18.x` and they scheduled for removal in Kedro `0.19.0`. To ensure a smooth transition, we strongly recommend you adopt the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) as soon as possible.
```

Reply:
Maybe that is 0.18.5?

Reply:
Done, it's 0.18.12.